packages/contracts-bedrock/src/L2/L1Block.sol

```diff
@@ -136,16 +121,31 @@ contract L1Block is ISemver, IGasToken {
         l1FeeScalar = _l1FeeScalar;
     }
 
-    /// @notice Updates the `isDeposit` flag and sets the L1 block values for an Isthmus upgraded chain.
-    ///         It updates the L1 block values through the `setL1BlockValuesEcotone` function.
-    ///         It forwards the calldata to the internally-used `setL1BlockValuesEcotone` function.
-    function setL1BlockValuesIsthmus() external {
-        // Set the isDeposit flag to true.
-        assembly {
-            sstore(IS_DEPOSIT_SLOT, 1)
-        }
+    /// @notice Updates the L1 block values for an Ecotone upgraded chain.
+    /// Params are packed and passed in as raw msg.data instead of ABI to reduce calldata size.
+    /// Params are expected to be in the following order:
+    ///   1. _baseFeeScalar      L1 base fee scalar
+    ///   2. _blobBaseFeeScalar  L1 blob base fee scalar
+    ///   3. _sequenceNumber     Number of L2 blocks since epoch start.
+    ///   4. _timestamp          L1 timestamp.
+    ///   5. _number             L1 blocknumber.
+    ///   6. _basefee            L1 base fee.
+    ///   7. _blobBaseFee        L1 blob base fee.
+    ///   8. _hash               L1 blockhash.
+    ///   9. _batcherHash        Versioned hash to authenticate batcher by.
```

Comment:
I opted to duplicate the NATSPEC on the public and the internal functions here. Lmk what do you think @skeletor_spaceman